packages/api/src/users/users.controller.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { INestApplication } from '@nestjs/common';
+import { CqrsModule } from '@nestjs/cqrs';
+import { Test } from '@nestjs/testing';
+
+import { UsersController } from './users.controller';
+import { CommandHandlers, EventHandlers } from './users.module';
+import { UsersRepository } from './users.repository';
+
+describe('Users controller', () => {
+  let app: INestApplication;
+  let usersController: UsersController;
+  let usersRepository: Partial<UsersRepository>;
+  const userId = 'Id';
+
+  describe('getMe', () => {
+    it('Should return user', async () => {
+      usersRepository.findById = jest.fn().mockResolvedValue({ id: userId });
```

Comment:
Now your test are aware of implementation details (you need to know that there is usersRepository and how they are implemented). I recommend to just interact with well defiend module API in test (use just COMMAND / EVENT / QUERY). Then you can refactor even usersRepository without breaking tests. Tests are not really valuable, if you need to change it while implementation details changes.